EasyReflectometry/calculators/bornagain/wrapper.py

```diff
@@ -1,7 +1,7 @@
 __author__ = 'github.com/arm61'
 
 import bornagain as ba
-from easyCore import np
+from easyscience import np
```

Comment:
I think we removed the `easycore` version of numpy and moved to
```python
import numpy as np
```

Reply:
Thanks